api-csharp/samples/start.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [ -z "$TRINSIC_ACCESS_TOKEN" ]; then
+    if [ ! -f .env ] || ! grep -q "TRINSIC_ACCESS_TOKEN" .env; then
+        echo "Error: TRINSIC_ACCESS_TOKEN environment variable not set or .env file not found"
+        exit 1
+    fi
+fi
+
+echo "Build UI sample..."
```

Comment:
C# also has a way to run it built-in to `.csproj` so if we choose to use native build systems we should aim to do that.